plutus-emulator/src/Wallet/Emulator/Servant.hs

```diff
@@ -0,0 +1,145 @@
+{-# LANGUAGE ConstraintKinds       #-}
+{-# LANGUAGE DeriveAnyClass        #-}
+{-# LANGUAGE DerivingStrategies    #-}
+{-# LANGUAGE FlexibleContexts      #-}
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE GADTs                 #-}
+{-# LANGUAGE LambdaCase            #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings     #-}
+{-# LANGUAGE TemplateHaskell       #-}
+{-# LANGUAGE TupleSections         #-}
+{-# LANGUAGE TypeApplications      #-}
+module Wallet.Emulator.Servant where
```

Comment:
This is really an emulator for the chain client, right? Not specifically for `Servant` or even necessarily for a server. So maybe let's put it in something like `Chain.Emulator`.

Reply:
In fact, I think clearly separating it out as the chain emulator interface would hopefully make it clearer how to piece it together with the overall "simulation emulator".